workflow/scripts/cluster_network.py

```diff
@@ -589,6 +598,13 @@ def plot_busmap_for_n_clusters(n, n_clusters, fn=None):
     else:
         n_clusters = int(snakemake.wildcards.clusters)
 
+    if transport_model:
+        logger.info(f"Using Transport Model.")
+        nodes_req = n.buses[f"{topological_boundaries}"].unique()
```

Comment:
The topological boundary should be `state` or `reeds_zone` according to the config? But columns here are `country` and `county`? Not sure if I am missing something?

Reply:
Changing to only list reeds_zone as elegible topological boundary